eureka/Dockerfile

```
@@ -0,0 +1,19 @@
+# Build stage
+FROM openjdk:17-jdk-slim AS builder
+
+WORKDIR /app
+
+COPY . .
+
+RUN chmod +x ./gradlew
+RUN ./gradlew clean build -x test
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)